pom.xml

```diff
@@ -202,7 +202,7 @@
                     <version>${jacoco-maven-plugin.version}</version>
                     <configuration>
                         <excludes>
-                            <exclude>**/*com/expediagroup/graphql/sample/**</exclude>
+                            <exclude>**/com/expediagroup/graphql/examples/**</exclude>
```

Comment:
Also updating the jacoco config: https://github.com/ExpediaGroup/graphql-kotlin/pull/376#discussion_r328146167